content/millennium/r4/care-provision/care-team.md

```diff
@@ -0,0 +1,140 @@
+---
+title: CareTeam | R4 API
+---
+
+# CareTeam
+
+* TOC
+{:toc}
+
+## Overview
+
+The R4 CareTeam resource can be used to represent care teams.
+
+The CareTeam resource represents a patient’s care team members. Care team members or participants include practitioners (physicians, nurses, technicians, etc.), family members, friends, guardians, and the patient. The care team can be specific to an encounter or to the patient across all encounters (longitudinal).
+
+The following fields are returned if valued:
+
+* [Care Team id](http://hl7.org/fhir/resource-definitions.html#Resource.id){:target="_blank"}
+* [Status (Active)](http://hl7.org/fhir/careteam-definitions.html#CareTeam.status){:target="_blank"}
+* [Category (Longitudinal or Encounter)](http://hl7.org/fhir/careteam-definitions.html#CareTeam.category){:target="_blank"}
+* [Name (Encounter-Level Only)](http://hl7.org/fhir/careteam-definitions.html#CareTeam.name){:target="_blank"}
+* [Subject](http://hl7.org/fhir/careteam-definitions.html#CareTeam.subject){:target="_blank"}
+* [Encounter (Encounter-Level Only)]( http://hl7.org/fhir/careteam-definitions.html#CareTeam.encounter){:target="_blank"}
+* [Participant](http://hl7.org/fhir/careteam-definitions.html#CareTeam.participant){:target="_blank"}
+  * [Role](http://hl7.org/fhir/careteam-definitions.html#CareTeam.participant.role){:target="_blank"}
+  * [Member](http://hl7.org/fhir/careteam-definitions.html#CareTeam.participant.member){:target="_blank"}
```

Comment:
Remove extra space

Reply:
Role and Member are subparts of Participant. Hence to differentiate from other components added the spaces. Should the spaces be removed?

Reply:
the spaces should stay